css/modals.css

```diff
@@ -0,0 +1,250 @@
+@import url("https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+html {
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: inherit;
+}
+
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  font-size: 1em;
+}
+
+#modalTemplate {
+  position: relative;
+  width: 100vw;
+  height: 100%;
+  background-color: white;
+  margin: 0;
+  padding: 0;
+  display: none;
+}
+
+#modalContent {
+  width: inherit;
+
+  /* display: flex; */
+  flex-direction: column;
+  align-items: start;
+  justify-content: start;
+  background-color: white;
+  text-align: left;
+  list-style: none;
+}
+
+#modalClose {
+  display: flex;
+  padding: 3%;
+  justify-self: right;
+  width: 10%;
+  margin: 0;
+  left: 90%;
+  position: absolute;
+  cursor: pointer;
+
+  /* font-weight: 500; */
+}
+
+#modalTitle {
+  font-size: 3em;
+  line-height: 1em;
+  font-family: "poppins", sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  text-align: left;
+  top: 10vh;
+  margin: 7% 0 0 0;
+  padding: 1em  0.8em;
+  justify-content: start;
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+#modalLanguages {
+  display: flex;
+  flex-direction: row !important;
+  list-style: none;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 1.2em;
+  font-family: "poppins", sans-serif;
+  font-weight: normal;
+  position: relative;
+  padding: 5%;
+}
+
+#badge1,
+#badge2,
+#badge3 {
+  display: flex;
+  padding: 1% 3%;
+  border: solid 3px #1a2236;
+  border-radius: 2px;
+}
+
+#modalBody {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: #1a2236;
+  font-size: 1em;
+  font-family: "Playfair Display", serif;
+  font-weight: normal;
+  background-color: white;
+  margin-block: 10%;
+  padding: 2%;
+  width: 100%;
+}
+
+#modalImage {
+  display: flex;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4%;
+  padding: 2%;
+  position: relative;
+}
+
+#modalDescription {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: 1%;
+  font-size: 1.5em;
+  margin: 5%;
+}
+
+/* .modal-footer {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  font-size: 0.5em;
+  font-family: "Playfair Display", serif;
+  font-weight: normal;
+  position: absolute;
+  padding: 1%;
+} */
```

Comment:
- [ ] Kindly remove the commented pieces of code as they make your work look messy.